core/embed/prodtest/README.md

```diff
@@ -240,6 +282,26 @@ FIRMWARE VERSION
 OK 0.2.6
 ```
 
+### BOOTLOADER_VERSION
+Returns the version of the bootlaoder.
+The command returns `OK` followed by the version in the format `<major>.<minor>.<patch>`.
+
+Example:
+```
+BOOTLOADER_VERSION
+OK 2.1.7
+```
+
+### FIRMWARE VERSION
+Returns the version of the boardloader.
```

Comment:
The command name is `BOARDLOADER VERSION`

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/b7964bb62095997f462700d08ca24db7143c35af